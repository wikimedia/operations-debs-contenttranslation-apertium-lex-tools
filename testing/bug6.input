^лабораторија<n><f><sg><nom><ind>/laboratory<n><sg><nom><ind>$ ^,<cm>/,<cm>$ ^вреден<adj><f><sg><nom><ind>/worth<adj><f><sg><nom><ind>/capable<adj><f><sg><nom><ind>/valuable<adj><f><sg><nom><ind>/industrious<adj><f><sg><nom><ind>/hard-working<adj><f><sg><nom><ind>$ ^околу<pr>/round<pr>/near<pr>/about<pr>/around<pr>$ 
