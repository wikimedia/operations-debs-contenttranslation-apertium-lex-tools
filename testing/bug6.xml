
<rules>
  <rule><match lemma="вреден"><select lemma="hard-working"/></match></rule>
</rules>
