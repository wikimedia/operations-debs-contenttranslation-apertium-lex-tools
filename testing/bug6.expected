^лабораторија<n><f><sg><nom><ind>/laboratory<n><sg><nom><ind>$ ^,<cm>/,<cm>$ ^вреден<adj><f><sg><nom><ind>/hard-working<adj><f><sg><nom><ind>$ ^околу<pr>/round<pr>/near<pr>/about<pr>/around<pr>$ 
